pkg/services/folder/model.go

```diff
@@ -61,8 +64,9 @@ type UpdateFolderCommand struct {
 // MoveFolderCommand captures the information required by the folder service
 // to move a folder.
 type MoveFolderCommand struct {
-	UID          string `json:"uid"`
-	NewParentUID string `json:"new_parent_uid"`
+	Folder       *Folder `json:"folder"`
```

Comment:
This isn't an objection, but I'm curious why we need the full folder object in the move command? And if we do need it, can we remove the `UID` field since it's in the Folder object?

Reply:
it's my lack of xorm knowledge/best practice, I am using Update for the Move, will maybe find a better way, or just use Update for the Move service